src/__tests__/pages/policy/input/ParameterEditor.test.js

```diff
@@ -0,0 +1,52 @@
+// CenteredMiddleColumn.test.js
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import CenteredMiddleColumn from "/Users/introvertedbot/Documents/Projects/open-source-work/policyengine-app/src/layout/CenteredMiddleColumn.jsx";
+
+describe("CenteredMiddleColumn", () => {
+  test("renders title", () => {
+    const { getByText } = render(<CenteredMiddleColumn title="Test Title" />);
+    expect(getByText("Test Title")).toBeInTheDocument();
+  });
+
+  test("renders description when provided", () => {
+    const { getByText } = render(
+      <CenteredMiddleColumn
+        title="Test Title"
+        description="Test Description"
+      />,
+    );
+    expect(getByText("Test Description")).toBeInTheDocument();
+  });
+
+  test("renders description label by default when description is provided", () => {
```

Comment:
I'm not sure I understand the difference between this test and the one above

Reply:
One is testing the actual description and another is testing the "description" text is present or not